### DISCUSSION TITLE
Changelog: Phantom 23.14.1

### DISCUSSION THREAD

**Discussion by adamdelphantom** [phantom/docs#113] [Announcements]
### **New Features & Improvements 🎉**

- 💰 **Ledger Support for Instant Sell:** You can now Instant Sell NFTs via your Ledger hardware wallet on mobile and extension!
- ⌨️ **Mobile Accessibility Improvements:** We made some mobile components more visible to accessibility tools.
- 🧪 **Added more Internal Automated Tests:** Ongoing work to automatically catch bugs!
- 👩🏽‍🍳 **Let us cook:** Much more building behind the scenes. Watch this space! 👀

### **Bug Fixes 🐛**

- 🔌 **dApp Connection:** Fixed a bug where iOS users were unable to connect to dApps in  Solana Only mode.
- 🪟 **iFrame Rendering:** Fixed a bug where some iOS users were unable to see iFrames in the Phantom mobile browser.
- 📔 **Account Switching:** Fixed a bug where switching accounts caused a visual glitch.
- 🪙 **Token Sending:** Removed stale token details after sending max balance.
- 🧾 **Account Addresses:** Fixed inaccurate displays of owned account addresses.
- 🔴 **Something Went Wrong:** Fixed a bug where users were seeing a “Something Went Wrong” error when viewing a collectible.
- 🔘 **Hidden Stake Button:** Fixed a bug where the stake button could be hidden on mobile.
- 🧊 **3D NFT Webview crash:** Fixed a bug where the webview would crash when viewing 3D collectibles on mobile.
- ↔️ **Swap Review Button Hidden:** Fixed a bug where the swap review button was hidden under the tab bar on mobile.
- 💰 **Sending No Value Tokens:** Fixed a bug where valueless tokens were displaying improper amounts during the send flow.
- 🏞️  **Unable to Save Photos:** Fixed a bug where users were unable to save photos on Android due to media library permissions.
- 📕 **Changing Accounts Opened Explore:** Fixed a bug where changing accounts in the mobile browser would navigate to the explore stack.
- 💵 **Fee Breakdown Detail:** We won’t show the total fees in duplicate when the swapper fee breakdown is available.

**Comment by C-o-d-e-C-o-w-b-o-y** [comment]
This fixed an `iFrame` issue Magic Eden was having on mobile web-view, thanks so much!!
